test/demuxer_test.exs

```diff
@@ -0,0 +1,132 @@
+defmodule Membrane.WebM.DemuxerTest do
+  use ExUnit.Case
+
+  import Membrane.Testing.Assertions
+
+  alias Membrane.Testing
+
+  @input_dir "./test/fixtures/"
+
+  defmodule TestPipeline do
+    use Membrane.Pipeline
+
+    @impl true
+    def handle_init(options) do
+      children = [
+        source: %Membrane.File.Source{
+          location: options.input_file,
+          chunk_size: 4096
+        },
+        demuxer: Membrane.WebM.Demuxer
+      ]
+
+      links = [
+        link(:source)
+        |> to(:demuxer)
+      ]
+
+      {{:ok, spec: %ParentSpec{children: children, links: links}},
+       %{output_dir: options.output_dir}}
+    end
+
+    @impl true
+    def handle_notification({:new_track, {track_id, track_info}}, :demuxer, _context, state) do
```

Comment:
Wouldn't it be possible to connect the branches for processing output tracks in handle_init?